docs/dev_guide/pr_review_procedure.rst

```diff
@@ -69,6 +65,13 @@ SunPy Maintainers
 This is the group of people who have push access to the main SunPy repository.
 The membership of this group is at the discretion of the Lead Developer, but shall generally be made up of people who have demonstrated themselves to be trust worthy and active contributors to the project.
 
+This group has `subgroups <https://github.com/orgs/sunpy/teams/sunpy-maintainers/teams>`__ for each section of the repository that has `maintainers <https://sunpy.org/team#maintainer-list>`__.
+The members of these groups will automatically be requested to review all PRs which change files in that subpackage.
```

Comment:
I would rather not do that at this point, it might be something we want to do later, but let's see how it goes. The idea of this is to get more people in the review pool not less...

fwiw GitHub does explicitly allow this as an option.